packages/mui-material/src/Pagination/Pagination.js

```diff
@@ -56,6 +60,12 @@ const Pagination = React.forwardRef(function Pagination(inProps, ref) {
     boundaryCount = 1,
     className,
     color = 'standard',
+    components = {
```

Comment:
Why do we need these in the `Pagination` component?

Reply:
@mnajdova, I realised I don't need it because renderItem does the trick, I'm gonna update the code.
Is a demo of icon customizations mandatory?